app/assets/stylesheets/colors.scss

```diff
@@ -0,0 +1,7 @@
+$green: #94B24D;
+$light_green: #D3FF82;
+$blue: #363D52;
+$light_blue: scale-color($blue, $lightness: +10%);
+$lighter_blue: scale-color($blue, $lightness: +20%);
```

Comment:
Name of variable `lighter_blue` should be written in all lowercase letters with hyphens instead of underscores